Gemfile.lock

```diff
@@ -1,32 +1,37 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    addressable (2.8.1)
-      public_suffix (>= 2.0.2, < 6.0)
-    builder (3.2.4)
+    addressable (2.8.7)
+      public_suffix (>= 2.0.2, < 7.0)
+    base64 (0.2.0)
+    builder (3.3.0)
     colorator (1.1.0)
-    concurrent-ruby (1.1.10)
+    concurrent-ruby (1.3.5)
+    csv (3.3.3)
     em-websocket (0.5.3)
       eventmachine (>= 0.12.9)
       http_parser.rb (~> 0)
     eventmachine (1.2.7)
-    ffi (1.15.5)
+    ffi (1.17.1)
     forwardable-extended (2.6.0)
-    google-protobuf (3.23.0)
+    google-protobuf (3.25.6)
     http_parser.rb (0.8.0)
-    i18n (1.12.0)
+    i18n (1.14.7)
       concurrent-ruby (~> 1.0)
-    jekyll (4.3.1)
+    jekyll (4.4.1)
       addressable (~> 2.4)
+      base64 (~> 0.2)
       colorator (~> 1.0)
+      csv (~> 3.0)
       em-websocket (~> 0.5)
       i18n (~> 1.0)
       jekyll-sass-converter (>= 2.0, < 4.0)
       jekyll-watch (~> 2.0)
+      json (~> 2.6)
       kramdown (~> 2.3, >= 2.3.1)
```

Comment:
can ones like this be simplified to
```suggestion
      kramdown (~> 2.3.1)
```

Reply:
I have no idea, tbh i don't even know what these symbols mean, i will read this and have a think https://blog.saeloun.com/2022/08/16/understanding-gemfile-and-gemfile-lock/#what-are-the-various-configuration-options-for-a-gem

Reply:
AIUI

`~>` means use this major version
so `~> 2.1` means use 2.1+ and less than 3.0

Reply:
https://blog.saeloun.com/2022/08/16/understanding-gemfile-and-gemfile-lock/#pessimistically-greater-than-or-equal-to-symbols

Reply:
yep, i see they are functionally the same, good to know. 

these are generated automatically though by `bundle update`. I just tried this change, then re-ran the update - sadly it ignores the change and re-adds the previous, so this would just make gem updates tedious

Reply:
I see i'm not locking the bundler version the Dockerfile uses, i've fixed this with `RUN gem install bundler:2.2.23`  so thanks for that - deleting the entire lock file and regenerating it gives the same result, so we don't actually know what version of ruby/bundler created this lockfile previously, another benefit of Docker